pyglet/math.py

```diff
@@ -1396,7 +1396,17 @@ def __repr__(self) -> str:
 
 
 class Quaternion(_typing.NamedTuple):
-    """Quaternion."""
+    """Quaternions are special 4-dimensional vectors useful for 3D rotations.
```

Comment:
I would avoid mentioning the Vec4 class entirely here. Quaternions are not really vectors, but a 4d number that is a type of complex number (as I understand it).